tuxemon/lib/compress_json.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
```

Comment:
is this directly pulled from his github?   i checked and its MIT licensed, so to be compliant, it will need a copy of it in the header https://en.wikipedia.org/wiki/MIT_License

Reply:
fixed